graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/QuerySuggestionsES6.java

```diff
@@ -55,7 +64,10 @@ public SuggestionResponse suggest(SuggestionRequest req) {
         final SearchSourceBuilder search = new SearchSourceBuilder()
                 .query(QueryBuilders.prefixQuery(req.field(), req.input()))
                 .size(0)
-                .aggregation(AggregationBuilders.terms("fieldvalues").field(req.field()).size(10));
+                .aggregation(AggregationBuilders.terms("fieldvalues").field(req.field()).size(10))
+                .suggest(new SuggestBuilder()
+                        .addSuggestion("corrections",
```

Comment:
typo? in ES6 class `corrections` in ES7 class `correction`

Reply:
This has to be just a unique name of the suggestion across the request and could be, theoretically, randomly generated. But to avoid any confusion, I have unified the name in both implementations. Thanks for the hint!